packages/components-core/src/components/RankingCard/index.js

```diff
@@ -58,44 +58,71 @@ CitationWrapper.propTypes = {
 
 const InfoLayerWrapper = ({ imageSize, body, heading, readMoreLink }) => {
   const [open, setOpen] = useState(false);
+  const uniqueIdRef = useRef(
+    `info-layer-${Math.floor(Math.random() * 100000)}`
+  );
 
   // TODO: test this
-  const handleButtonClick = async () => {
-    await setOpen(!open);
-    trackGAEvent({
-      ...gaDefaultObject,
-      text: "Expand ranking",
-      action: open ? AVAILABLE_GA_ACTIONS.OPEN : AVAILABLE_GA_ACTIONS.CLOSE,
-      section: heading,
-    });
+  const handleButtonClick = event => {
+    if (event.type === "click" || event.key === "Enter" || event.key === " ") {
+      setOpen(!open);
+      trackGAEvent({
+        ...gaDefaultObject,
+        text: "Expand ranking",
+        action: open ? AVAILABLE_GA_ACTIONS.OPEN : AVAILABLE_GA_ACTIONS.CLOSE,
+        section: heading,
+      });
+    }
   };
 
   return (
     <div
       className={classNames("info-layer", { [`active`]: open })}
       data-testid="info-layer"
+      id={uniqueIdRef.current}
     >
       <div className="content">
-        <div className="header">
-          {isSmallSize(imageSize) ? (
+        <div
+          className={classNames("header", {
+            [`closed`]: isSmallSize(imageSize) && !open,
+          })}
+        >
+          {isSmallSize(imageSize) && (
             // eslint-disable-next-line react/no-danger
             <p dangerouslySetInnerHTML={sanitizeDangerousMarkup(body)} />
-          ) : (
-            <h4>{heading}</h4>
           )}
-          <button
-            onClick={handleButtonClick}
-            id="dispatch"
-            className="btn btn-expand"
-            aria-label="Expand ranking"
-            type="button"
-            aria-expanded="false"
-            data-toggle="collapse"
-            data-target="#collapseExample"
-          >
-            <i className="fas fa-chevron-up" />
-            <span className="visually-hidden">Expand</span>
-          </button>
+          {!isSmallSize(imageSize) && (
+            <>
+              <button
+                onClick={handleButtonClick}
+                id="dispatch"
+                className="btn-expand"
+                aria-label="Expand ranking"
+                type="button"
+                aria-expanded={open}
+                aria-controls={uniqueIdRef.current}
+              >
+                <h4>{heading}</h4>
+              </button>
+              <i className="fas fa-chevron-up" />
```

Comment:
The `fa-chevron-up` `i` tag should go inside the button, like with the small size... as it is here, the chevron isn't clickable on the larger version and that's problematic.

Reply:
![Screenshot 2024-06-10 at 3 30 34 PM](https://github.com/ASU/asu-unity-stack/assets/212131/a06df05c-25a2-4b77-8409-563d096b23ff)